### PR TITLE
refactor: Created Unified Login/Register Component for User and Admin Portals

### LINKED ISSUE #1377: Refactor: Use the same Login/Register Component on User and Admin
**Is your feature request related to a problem? Please describe.**
The user and admin currently have their respective Login and Register Components
The user portal's UI for Login and Register Component does not look similar to what we have in admin portal
This creates unnecessary dependency in our project where we always need to synchronise the UI and other elements for both the User portal and Admin portal.

**Describe the solution you'd like**
Keeping the logic intact for authentication of user and redirection to respective User Portal or Admin Panel, create a single Login/Register Component made from the base UI and code of Admin Panel's Login/Register Component.

**Describe alternatives you've considered**
NA

**Approach to be followed (optional)**
NA

**Additional context**
Follow this PR, as it implements tab based navigation b/w User Login and Admin Login (However the components are different and they should be one), make sure that you are also aware of the current changes that are being made to Login/Register Component.

https://github.com/PalisadoesFoundation/talawa-admin/pull/1371

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by akhilender-bongirwar**
**What kind of change does this PR introduce?**
Resolves the problem of having separate Login and Register Components for User and Admin portals, causing UI inconsistencies.

**Issue Number:**

Fixes #1377 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**

https://github.com/PalisadoesFoundation/talawa-admin/assets/112749383/fbefe7bd-a11a-4dbf-98be-90b8b2492736

**If relevant, did you update the documentation?**
Not relevant

**Summary**

- Created a shared Login/Register Component that can be used seamlessly across both User and Admin portals.
- Ensured consistency in UI elements to eliminate the need for synchronization between the portals.
- Maintained existing authentication logic and redirection functionality.
- Thoroughly tested the unified component to validate its integration and verify the intended behavior for both portals.
- Also increased margin and padding in the Login and register page to match the figma file

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by Cioppolo14** [comment]
@noman2002 @tasneemkoushar Can you review this PR?

File: src/App.tsx [hunk 1]
```diff
@@ -115,7 +114,7 @@ function app(): JSX.Element {
         <Route exact path="/forgotPassword" component={ForgotPassword} />
 
         {/* User Portal Routes */}
-        <Route exact path="/user" component={UserLoginPage} />
+        <Route exact path="/user" component={LoginPage} />
```
Comment (rishav-jha-mech) [inline src/App.tsx:117]: @akhilender-bongirwar you can remove this path, one single path for the Login Page is sufficient.

Comment (akhilender-bongirwar) [reply src/App.tsx:117]: @rishav-jha-mech, in the `LoginPortalToggle` component, we are switching to the `/user` route. I have utilized it to set the role as the default user when the route is `/user`. Currently, it is necessary for the functionality.

Comment (akhilender-bongirwar) [reply src/App.tsx:117]: However, do you want me to remove the `LoginPortalToggle` component and instead add it to the Login Page? This way, we can switch roles within a single path for the Login Page.

Comment (rishav-jha-mech) [reply src/App.tsx:117]: @akhilender-bongirwar a new endpoint is not required for that, you can remove it, by default let admin be selected for now

Comment (akhilender-bongirwar) [reply src/App.tsx]: @rishav-jha-mech  I apologize for the delayed reply, as I had mid-semester exams. I have made the necessary changes to remove the `/user` endpoint and set admin as the default selection. Please review : )

**Review by noman2002 — APPROVED** [review]
Changes looks fine to me. 
@rishav-jha-mech Please have a look.

**Review by rishav-jha-mech — APPROVED** [review]

**Comment by akhilender-bongirwar** [comment]
@noman2002 @rishav-jha-mech why was the `codecov/project` was failing even though I have covered everything with tests?

**Comment by palisadoes** [comment]
It's a negligible fall -0.06%

**Comment by akhilender-bongirwar** [comment]
> It's a negligible fall -0.06%

Sir, I don't understand what might be the reason for the fall.

**Pull request merged**